installation/generate_helm_values.py

```diff
@@ -302,11 +302,19 @@ def iter_helm_values() -> collections.abc.Generator[tuple[str, dict], None, None
         delivery_db_backup_enabled = 'deliveryDbBackup' in extensions
         freshclam_enabled = 'clamav' in extensions
 
+        delivery_service_cfg = cfg_set.delivery_service()
+
+        # inject rescoring cfg from delivery-service cfg to avoid duplication
+        rescoring_rule_sets = delivery_service_cfg.features_cfg()['rescoring']
```

Comment:
Also, please consider the case when there is no rescoring cfg specified (right now this would cause a `KeyError`).